Cargo.toml

```diff
@@ -17,14 +17,14 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write" } # git tag -s $(git describe --dirty)
-zcash_client_backend = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write", features = ["lightwalletd-tonic", "orchard", "transparent-inputs"] }
-zcash_encoding = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write" }
-zcash_keys = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write", features = ["orchard"] }
+zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", rev = "84469ad71a3f389aaf839e2d4c40e612fa8b64a0" }
```

Comment:
How do we feel about having something more human-meaningful than a commit hash?   I feel like to merge into `dev` it ought to be a tag.